project/bhchen2001/README.rst

```diff
@@ -0,0 +1,95 @@
+========================
+Parallel Vonoroi Diagram
```

Comment:
It's Voronoi diagram, not "Vonoroi": https://mathworld.wolfram.com/VoronoiDiagram.html